### PR TITLE
ATACseq

### PR DESCRIPTION
**Pull request by lldelisle**
Here is my ATAC-seq workflow.
There are some awk steps which are not so 'galactic'... If someone is brave enough to convert them, feel free.

### CODE REVIEW

File: workflows/epigenetics/ATACseq/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+- The amount of reads 500bp from summits and the total number of reads are computed if further normalization is wanted.
+- Other QC are performed:
+  - A histogram with fragment length is computed.
+  - The evaluation of percentage of reads to chrM or MT is computed.
+- A multiQC is run to have an overview of the QC.
+
+### Warning
+
+- The coverage output is not normalized.
+- We are aware that the fact that a common reference file is used for bowtie2 and bedtools slopbed reduces the number of reference possible. Talk to your administrator if this brakes your research.
```
Comment (mvdbeek) [inline workflows/epigenetics/ATACseq/README.md:30]:
```suggestion
- The `reference_genome` parameter value is used to select references in bowtie2 and bedtools slopbed. Only references that are present in bowtie2 **and** bedtools slopbed are selectable. If your favorite reference genome is not available ask your administrator to make sure that each bowtie2 reference has a corresponding len file for use in bedtools slopbed.
```

**Review by mvdbeek — APPROVED** [review]
This is very exciting! Do you want to also add a sentence to link to the training materials (https://training.galaxyproject.org/training-material/topics/epigenetics/tutorials/atac-seq/slides.html#1, https://training.galaxyproject.org/training-material/topics/epigenetics/tutorials/atac-seq/tutorial.html) for background information?

**Comment by lldelisle** [comment]
@simleo 
Should this `.workflowhub.yml` added by the contributor or you prefer to add it yourself?

**Comment by mvdbeek** [comment]
I'll add this to planemo, but no problem if you add this yourself for now.

**Comment by simleo** [comment]
@lldelisle that file is used by `gen_crates.py` to register the workflow with LifeMonitor. At the moment this process is not yet automated, so I have to keep track of newly added workflows anyway :slightly_smiling_face:

So don't worry about this, it's not a big deal for me to add those files. In case you want to add them yourself, the format is simply:

```yaml
version: '0.1'
registries:
- url: https://workflowhub.eu
  project: iwc
  workflow: WORKFLOW_REPOSITORY_NAME/DOCKSTORE_WORKFLOW_NAME
```

Where `WORKFLOW_REPOSITORY_NAME` is the name of the repository where the workflow will be published under https://github.com/iwc-workflows, i.e., the basename of its parent directory here in this repo (in this case, `ATACseq`); `DOCKSTORE_WORKFLOW_NAME` is the value of the `name` field for the workflow in `.dockstore.yml` (in this case, `main`). The rest does not change (all workflows end up on https://workflowhub.eu under the `iwc` project).

**Comment by mvdbeek** [comment]
Yes, that would be good!

**Comment by mvdbeek** [comment]
This is all so cool!

**Comment by lldelisle** [comment]
Great!

**Pull request merged**